diff --git a/vendor/plugins/acts_as_solr/lib/solr/request/standard.rb b/vendor/plugins/acts_as_solr/lib/solr/request/standard.rb
index 03ec243..18ec2d6 100755
--- a/vendor/plugins/acts_as_solr/lib/solr/request/standard.rb
+++ b/vendor/plugins/acts_as_solr/lib/solr/request/standard.rb
@@ -50,7 +50,9 @@ class Solr::Request::Standard < Solr::Request::Select
       key = sort.keys[0]
       "#{key.to_s} #{sort[key] == :descending ? 'desc' : 'asc'}"
     end.join(',') if @params[:sort]
-    hash[:q] = sort ? "#{@params[:query]};#{sort}" : @params[:query]
+    #hash[:q] = sort ? "#{@params[:query]};#{sort}" : @params[:query]
+    hash[:sort] = sort
+    hash[:q] = @params[:query]
     hash["q.op"] = @params[:operator]
     hash[:df] = @params[:default_field]
 
